src/longbow/utils/model.py

```diff
@@ -2174,5 +2174,96 @@ def build_pre_configured_model(model_name):
+                ("L", "TPV2_adapter", "cDNA", "Poly_A", "idx", "rev_bind"),
+                ("M", "TPV2_adapter", "cDNA", "Poly_A", "idx", "rev_bind"),
+                ("N", "TPV2_adapter", "cDNA", "Poly_A", "idx", "rev_bind"),
+                ("O", "TPV2_adapter", "cDNA", "Poly_A", "idx", "rev_bind", "P"),
+            ),
+            "adapters": {
+                "TPV2_adapter": "CTACACGACGCTCTTCCGATCTTGGATTGATATGTAATACGACTCACTATAG",
+                "rev_bind": "CTCTGCGTTGATACCACTGCTT",
+                "A": "AGCTTACTTGTGAAGAT",
+                "B": "ACTTGTAAGCTGTCTAT",
+                "C": "ACTCTGTCAGGTCCGAT",
+                "D": "ACCTCCTCCTCCAGAAT",
+                "E": "AACCGGACACACTTAGT",
+                "F": "AGAGTCCAATTCGCAGT",
+                "G": "AATCAAGGCTTAACGGT",
+                "H": "ATGTTGAATCCTAGCGT",
+                "I": "AGTGCGTTGCGAATTGT",
+                "J": "AATTGCGTAGTTGGCCT",
+                "K": "ACACTTGGTCGCAATCT",
+                "L": "AGTAAGCCTTCGTGTCT",
+                "M": "ACCTAGATCAGAGCCTT",
+                "N": "AGGTATGCCGGTTAAGT",
+                "O": "AAGTCACCGGCACCTTT",
+                "P": "ATGAAGTGGCTCGAGA",
+                "Poly_A": {HPR_SEGMENT_TYPE_NAME: ("A", 30)},
+                "idx": {FIXED_LENGTH_RANDOM_SEGMENT_TYPE_NAME: 10},
+                "cDNA": RANDOM_SEGMENT_NAME,
+            },
+            "direct_connections": {
+                "A": {"TPV2_adapter"},
+                "B": {"TPV2_adapter"},
+                "C": {"TPV2_adapter"},
+                "D": {"TPV2_adapter"},
+                "E": {"TPV2_adapter"},
+                "F": {"TPV2_adapter"},
+                "G": {"TPV2_adapter"},
+                "H": {"TPV2_adapter"},
+                "I": {"TPV2_adapter"},
+                "J": {"TPV2_adapter"},
+                "K": {"TPV2_adapter"},
+                "L": {"TPV2_adapter"},
+                "M": {"TPV2_adapter"},
+                "N": {"TPV2_adapter"},
+                "O": {"TPV2_adapter"},
+                "TPV2_adapter": {"cDNA"},
+                "cDNA": {"Poly_A"},
+                "Poly_A": {"idx"},
+                "idx": {"rev_bind"},
+                "rev_bind": {
+                    "A",
+                    "B",
+                    "C",
+                    "D",
+                    "E",
+                    "F",
+                    "G",
+                    "H",
+                    "I",
+                    "J",
+                    "K",
+                    "L",
+                    "M",
+                    "N",
+                    "O",
+                    "P",
+                },
+            },
+            "start_element_names": {"A", "TPV2_adapter"},
+            "end_element_names": {"P", "rev_bind"},
+            "named_random_segments": {"idx", "cDNA"},
+            "coding_region": "cDNA",
+            "annotation_segments": {
+                "idx": [(longbow.utils.constants.READ_INDEX_TAG)],
```

Comment:
This tuple needs a second entry with the tag in which to store the position information for the `READ_INDEX_TAG`.

You can probably just reuse `longbow.utils.constants.READ_BARCODE_POS_TAG`.

Reply:
Done